### PR TITLE
Add stylesheet & update DOM structure

### PR DESCRIPTION
**Pull request by nwingt**
Depends on #1

### CODE REVIEW

File: src/sdk.js [hunk 1]
```diff
@@ -1,4 +1,26 @@
-const elements = document.querySelectorAll('div.likecoin-button-like');
+const style = document.createElement('style');
+style.innerHTML = `
```
Comment (williamchong007) [inline src/sdk.js]: not sure if we have better way to do this instead of `style.innerHTML`

Comment (nwingt) [reply src/sdk.js]: Maybe we just minimize the CSS for now? We can parse SCSS/Stylus in Webpack later

Comment (williamchong007) [reply src/sdk.js]: Is using [insertRule](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule) too obsolete, or is there something similar in modern es?

Comment (nwingt) [reply src/sdk.js]: That looks primitive. Maybe need to study other SDK for CSS injection

**Pull request merged**